test/acs-engine-test/report/report.go

```diff
@@ -97,14 +119,16 @@ func (h *ReportMgr) Copy() *ReportMgr {
 	return n
 }
 
-func (h *ReportMgr) Process(txt, location string) {
-	for key, regex := range errorRegexpMap {
-		if match, _ := regexp.MatchString(regex, txt); match {
-			h.addFailure(key, map[string]int{location: 1})
+func (h *ReportMgr) Process(txt, testName, location string) {
+	for _, logErr := range logErrors {
+		if match, _ := regexp.MatchString(logErr.regex, txt); match {
```

Comment:
Irrespective (even know when text is changing) no MDM metric should be dependent on random error strings to determine failure/success. This method is error prone. and non deterministic and I wouldn't want it to be used in the RP.